test/e2e/specs/wp-my-home-support-search-spec.js

```diff
@@ -78,48 +61,59 @@ describe( `[${ host }] My Home "Get help" support search card: (${ screenSize })
 		);
 	} );
 
-	step( 'Verify Inline Help support search is not displayed on My Home', async function () {
-		// The toggle only hides once the "Get help" card is rendered so
-		// we need to give it a little time to be removed.
-		await inlineHelpPopoverComponent.waitForToggleNotToBePresent();
-
-		// Once removed we can assert is is invisible.
-		const isToggleVisible = await inlineHelpPopoverComponent.isToggleVisible();
-		assert.equal(
-			isToggleVisible,
-			false,
-			'Inline Help support search was not correctly hidden on Home page.'
-		);
-	} );
-
 	step( 'Displays contextual search results by default', async function () {
 		supportSearchComponent = await SupportSearchComponent.Expect( driver );
-		const resultsCount = await supportSearchComponent.getSearchResultsCount();
-		assert.equal( resultsCount, 5, 'There are no contextual results displayed' );
+		const resultsCount = await supportSearchComponent.getDefaultResultsCount();
+		assert.equal( resultsCount, 5, 'The default results are displayed' );
```

Comment:
> 'The default results are displayed

This is going to be the error message if the assertion fails. I wonder whether it reads enough like an error.

Ideally, it would describe _what went wrong_ rather than what happened. 

Small detail but will make debugging these tests much easier if/when they break (which is why I wrote them in the first place) 😄

Reply:
> Ideally, it would describe what went wrong rather than what happened.

I hadn't considered the error messages from that perspective. That's really helpful to keep in mind. I had changed it because I thought, "This assertion doesn't determine that no contextual results are displayed, I should change it to describe what it does determine." 😅